test/mpi/f08/pt2pt/Makefile.am

```diff
@@ -26,7 +26,8 @@ noinst_PROGRAMS = greqf08 \
                   prsendf08 \
                   pssendf08 \
                   sendrecvf08 \
-                  sendrecvreplf08
+                  sendrecvreplf08 \
+		  pt2pt_largef08
```

Comment:
should add to `.gitignore`, too

Reply:
You mean `testlist`?

Reply:
both 😄. we are not very good about it, but all test binaries should be added to `test/mpi/.gitignore` to avoid git complaining if you do `make` in the testsuite and end up with a bunch of untracked binary files.

Reply:
I added that to `testlist` and the tests passed. Since none of the testing binaries are in `.gitignore`  currently, I don't think I should start adding it now. I believe `make clean` is able to clean out the compiled binaries anyway.

Reply:
To be clear, many test binaries are listed in `test/mpi/.gitignore`. Definitely not none. We can clean that up separately, though.

Reply:
Just realized that you are referring to `test/mpi/.gitignore`.  Didn't realize `git` let you do that 😄 Yeah, we'll clean it up separately.